x-pack/platform/plugins/shared/fleet/common/constants/routes.ts

```diff
@@ -50,6 +50,7 @@ export const EPM_API_ROUTES = {
   STATS_PATTERN: `${EPM_PACKAGES_MANY}/{pkgName}/stats`,
   BULK_ASSETS_PATTERN: `${EPM_API_ROOT}/bulk_assets`,
   INPUTS_PATTERN: `${EPM_API_ROOT}/templates/{pkgName}/{pkgVersion}/inputs`,
+  INPUT_PACKAGES_PATTERN: `${EPM_API_ROOT}/input_packages/{pkgName}/{pkgVersion}`,
```

Comment:
nit: on the API route, it may be interesting to have something that tell it's about assets, also it may interesting to not restrict it to the input package at least in the route name, so in the future we can implement this for integration package see https://github.com/elastic/kibana/issues/160775

Something like
```
/EPM_API_ROOT/packages/{name}/version/datastream_assets
```